web/src/main/resources/org/akaza/openclinica/i18n/page_messages.properties

```diff
@@ -924,20 +924,6 @@ you_are_not_owner_some_items_cannot_delete = You are not the owner of some items
 
 you_are_trying_resolve_discrepancy_not_exist = You are trying to resolve a Discrepancy Note that does not exist.
 
-you_can_access_odm_12 = You can access your ODM 1.2 w/OpenClinica Extension XML file <a href='AccessFile?fileId={0}'>here</a>.<br/>
-
-you_can_access_odm_12_xml = You can access your ODM 1.2 XML file <a href='AccessFile?fileId={0}'>here</a>.<br/>
-
-you_can_access_odm_13 = You can access your ODM 1.3 XML file <a href='AccessFile?fileId={0}'>here</a>.<br/>
-
-you_can_access_odm_13_xml = You can access your ODM 1.3 w/OpenClinica Extension XML file <a href='AccessFile?fileId={0}'>here</a>.<br/>
-
-you_can_access_spss = You can access your SPSS files <a href='AccessFile?fileId={0}'>here</a>.<br/>
-
-you_can_access_tab_delimited = You can access your tab-delimited file <a href='AccessFile?fileId={0}'>here</a>.<br/>
-
-you_can_review_the_data = <p>You can review the entered data <a href='{0}ListStudySubjects'>here</a>.
-
```

Comment:
- I found a duplicated string for you_can_access_tab_delimited (line 937 and 1050) and for you_can_review_the_data (line 339 and 1046), that why i remove them. Please check this again.